src/components/common/SmallTextEditor.vue

```diff
@@ -0,0 +1,44 @@
+<template>
```

Comment:
@poojahpatil90 , I feel like adding a new component would increase maintenance complexity in the future. Eg: if we'd like to update style, or changing the way we use validation component, etc.

As the difference dwell in template only, I suggest we keep using the old `TextEditor` component:
- solution1: add [rows attribute](https://www.w3schools.com/tags/att_textarea_rows.asp) to text-area to handle the height
- solution2: make a conditional render using a new added prop `rows` if we really want an `<input>` tag. Pseudo code:
```
<text-area v-if='rows > 1' :rows="rows" .... />
<input v-else ... />
```

What do you think ?

Reply:
This is a good idea!  I have made the change. Made rows to have a default value of 1.